trafficlight_recognizer/launch/feat_proj.launch

```diff
@@ -13,5 +13,4 @@
     <param name="roi_search_min_distance" value="$(arg roi_search_min_distance)"/>
     <param name="roi_search_max_distance" value="$(arg roi_search_max_distance)"/>
   </node>
-  
```

Comment:
This line change is no necessary, would prefer if this change is not in this PR.